feat: how to get new API tokens (#312)
Substra/substra-documentation: docs/source/documentation/api_tokens_generation.rst

How-to generate new API tokens for login
================================

This short guide explains how to use the API tokens management page.

Generating new API tokens
-----------------------------------------

> To do so you need to go to the API tokens management page on the web application. Then you will see a list of your current token as well

Reviewer: ```suggestion
To do so you need to go to the API tokens management page on the web application. You will see a list of your current token as well
```

as an option to generate a new one. 

> Clicking on this button will open a menu allowing you to pick a name and an expiration date for

Reviewer: ```suggestion
Clicking on this button opens a menu allowing you to pick a name and an expiration date for
```

your new token. 

> Afterward you will be able to see your token only once. Do copy it somewhere safe before proceeding with your work. 

Reviewer: ```suggestion
Afterward your token will be shown only once. Do copy it somewhere safe before proceeding with your work. 
```
